backend/app/api/endpoints/admin/tasks.py

```
@@ -0,0 +1,84 @@
+    # Verify user exists
+    target_user = user_service.get_user_by_name(db, user_name)
+
+    # Create task
+    return task_kinds_service.create_task_or_append(
+        db=db, obj_in=task, user=target_user, task_id=task_id
+    )
+
+
+@router.post("/generate-admin-token", response_model=Token)
+async def generate_admin_token(
+    db: Session = Depends(get_db), current_user: User = Depends(get_admin_user)
```

Ruff (0.14.13)

Unused function argument: `db`

(ARG001)